Makefile

```diff
@@ -33,7 +33,7 @@ IMAGE_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD | sed 's/\///g')
 GIT_REF = $(shell git rev-parse --short=8 --verify HEAD)
 
 BUILDMNT = /go/src/$(GOTARGET)
-BUILD_IMAGE ?= golang:1.8
+BUILD_IMAGE ?= gcr.io/heptio-images/golang:1.9-alpine3.6
```

Comment:
I feel like this change should have a little more pomp and circumstance than it does. Maybe a note in the changelog about "targetting Go 1.9 now?"